android-automation/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.8
+
+RUN apk add --update
```

Comment:
Not really needed most of the time, and if it's needed for some reason it's better to just merge it with the next layer, as it will result in a smaller image

Reply:
Get rid of apk add --update